mautic/mautic
# Update GrapesJs to latest version (Fixes #10629)

#### Description:

Updates GrapeJS to latest version, which fixes padding problems on mjml (Fixes [#10629](https://github.com/mautic/mautic/issues/10629)).

#### Steps to test this PR:

1. Edit an e-mail with an mjml template
2. Select an element and change the padding of the element in the style manager
3. Select another element
4. Re-select the first element and check if the padding show in the style manager is correct


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10630"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
